src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowEurostagTutorialExample1Test.java

```diff
@@ -465,4 +465,26 @@ void maxOuterLoopIterationTest() {
         assertFalse(result.isFullyConverged());
         assertEquals(LoadFlowResult.ComponentResult.Status.MAX_ITERATION_REACHED, result.getComponentResults().get(0).getStatus());
     }
+
+    @Test
+    void testWriteReadSlackBus() {
+        Network network = EurostagFactory.fix(EurostagTutorialExample1Factory.create());
+        network.getVariantManager().cloneVariant(network.getVariantManager().getWorkingVariantId(), "newVariant");
+        LoadFlowParameters parameters = new LoadFlowParameters();
+        parameters.setWriteSlackBus(true);
```

Comment:
you could use method chaining, and also setReadSlackBus(true) to better emphasize the test intention